src/BSD-2-Clause-NetBSD.xml

```diff
@@ -1,18 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <SPDXLicenseCollection xmlns="http://www.spdx.org/license">
    <license isOsiApproved="false" licenseId="BSD-2-Clause-NetBSD"
-            name="BSD 2-Clause NetBSD License">
+            name="BSD 2-Clause NetBSD License"
+            isDeprecated="true" deprecatedVersion="3.9">
+      <obsoletedBys>
+        <obsoletedBy>BSD-2-Clause</obsoletedBy>
+      </obsoletedBys>
       <crossRefs>
          <crossRef>http://www.netbsd.org/about/redistribution.html#default</crossRef>
       </crossRefs>
-      <notes>NetBSD adopted this 2-clause license in 2008 for code contributed to NetBSD Foundation.</notes>
+      <notes>
+        NetBSD adopted this 2-clause license in 2008 for code contributed to NetBSD Foundation.
+        It is being deprecated as a separate license because the only difference from BSD-2-Clause
+        is the line describing that the code is derived from software contributed to NetBSD,
+        which is not viewed as being a substantive part of the license text.
+      </notes>
     <text>
       <copyrightText>
          <p>Copyright (c) 2008 The NetBSD Foundation, Inc. All rights reserved.</p>
       </copyrightText>
 
       <p>This code is derived from software contributed to The NetBSD Foundation by
-        <alt match=".*" name="contributors"/>
+        <alt match=".*" name="contributors">[contributor name]</alt>.
```

Comment:
I think we may want to leave this with the incomplete sentence to be compatible with the "official" text at https://www.netbsd.org/about/redistribution.html.  There has been a lot of issues raised on the license list publisher, the license list data, and the license list XML that we don't retain the canonical text.  Making this change will deviate from the canonical text.

Reply:
fair enough! thanks @goneall, I'll update the PR to revert this part.